vuepress/docs/next/tutorials/solution/customer-portal.md

```diff
@@ -22,36 +22,40 @@ This tutorial covers:
-* Use the [Entando CLI](../../docs/getting-started/entando-cli.md) command `ent check-env develop` to verify all dependencies.
-* A Jira Service Management account.
-* A [Keycloak server](https://www.keycloak.org/docs/15.0/server_admin/#_email) configured with the SMTP server settings to enable email verification for users. 
+* Use the [Entando CLI](../../docs/getting-started/entando-cli.md) command `ent check-env develop` to verify dependencies
+* A Jira Service Management account
+* A [Keycloak server](https://www.keycloak.org/docs/15.0/server_admin/#_email) configured with the SMTP  settings to enable email verification for users
 
 ### Automatic Install via the Entando Hub
-Install the Customer Portal in two steps by integrating the Entando Hub into your App Builder. 
-1. Log into your `App Builder` → `Hub` → `Select Registry` → choose `Entando Hub` if it has been configured. 
-     1. If not, choose `New Registry`
-     2. In the pop-up window, enter `Entando Hub` and `https://entando.com/entando-hub-api/appbuilder/api` for the URL, then `Save`
-     3. Click on the Hub in the Registry 
-
-2. From the Hub Catalog, `Deploy` and `Install` the Customer Portal application first, then the content bundle.
-
-3. Set up permissions to configure the service:
-   - [Login to your Keycloak instance](../../docs/consume/identity-management.md#logging-into-your-keycloak-instance) as an admin.
-   - Give at least one user the ability to manage the Customer Portal by granting the `cp-admin` role. Assign the `cp-admin` role for the `pn-a71d68dd-166dc0f4-entandodemo-customerportal-server` client. See [Role Assignment in ID Management](../../docs/consume/identity-management.md#authorization) for more details.
-   - Give the generated plugin client permission to manage users. 
-       1. From the left sidebar, go to `Clients` and select client ID `pn-a71d68dd-166dc0f4-entandodemo-customerportal-server`. 
-       2. Click the `Service Account` tab at the top of the page and select `realm-management` from the `Client Roles` field. 
-       3. Choose `realm-admin` from `Available Roles`. Click `Add selected`. It should appear as an `Assigned Role`.
-
-4. To navigate to your CP: 
-   * From the sidebar →  `Page` → `Management` 
-   * Find the `Customer Portal` folder
-   * From the `Actions` pull-down menu →  `View Published Page`
+
+#### Integrate the Entando Hub into your App Builder
+1. Log in to your `App Builder` and go to `Hub` → `Select Registry`
+2. Choose `Entando Hub` if it has been configured, otherwise:
+   1. Choose `New Registry`
+   2. In the pop-up window, enter `Entando Hub` and `https://entando.com/entando-hub-api/appbuilder/api` for the URL, then `Save`
+   3. Click on the Hub in the Registry
+
+#### Install the Customer Portal
+From the Hub Catalog, `Deploy` and `Install` the Customer Portal application first, then the content bundle
+
+#### Set up permissions to configure the service
```

Comment:
i was going to leave these with the capitalization of regular sentences, but don't mind using header capitalization; the only reason i was doing headers instead of bold was so that i could reference a specific subsection